packages/server/server/api/destinations/destination.controller.ts

```diff
@@ -15,9 +16,10 @@ class DestinationController implements Controller {
     }
 
     private initializeRoutes() {
-        this.router.post(this.path, this.createDestination.bind(this));
-        this.router.get(this.path, this.getDestinationsByTrecipeId.bind(this));
-        this.router.get(`${this.path}/:id`, this.getDestinationById.bind(this));
+        this.router.post(this.path, passportAuth, this.createDestination.bind(this));
+        this.router.get(this.path, this.getDestinationByPlaceId.bind(this));
+        this.router.get(`${this.path}/in`, this.getDestinationsByTrecipeId.bind(this));
+        this.router.get(`${this.path}/:id`, passportAuth, this.getDestinationById.bind(this));
```

Comment:
I am 99.9% sure this is the wrong line LOL

Reply:
Andddd now you got your 100%. Sorry, was a long day 🤦